src/main/java/com/thevoxelbox/voxelsniper/brush/type/AbstractBrush.java

```diff
@@ -61,7 +72,7 @@ public void handleCommand(String[] parameters, Snipe snipe) {
      * @param index       Parameter index
      * @return Sorted completions.
      */
-    public List<String> sortCompletions(Stream<String> completions, String parameter, int index) {
+    protected List<String> sortCompletions(Stream<String> completions, String parameter, int index) {
```

Comment:
Is there a reason why all methods have been changed to protected in this class? Wouldn't that result in everyone using the API no longer being able to use these methods (unless they aren't considered API).

Reply:
What did I do 🤯 ty